src/components/pages/landing-de.tsx

```diff
@@ -1,203 +1,225 @@
+            'text-center text-4xl',
+            'tracking-tight font-bold',
+            'max-w-2xl mt-7 mx-auto px-2'
+          )}
+        >
+          <p className="mb-8">
+            Unsere Lernplattform wird von einem gemeinnützigen Verein
+            entwickelt. Sie ist komplett{' '}
+            <b className="font-extrabold">
+              kostenlos, werbefrei und{' '}
+              <span className="whitespace-nowrap">frei lizenziert</span>
+            </b>
+            .{' '}
+            <Underlined
+              className="font-handwritten text-brand"
+              style={{ fontSize: '1.2em' }}
+            >
+              Für immer!
+            </Underlined>
           </p>
-          <Button href="/beitreten">
-            Mitglied werden{' '}
-            <FontAwesomeIcon icon={faArrowCircleRight} size="1x" />
-          </Button>
-        </FlexCol>
-        <FlexCol>
-          <p className="serlo-p ml-0">
-            Softwareentwicklung und Lerninhalte erstellen kostet Geld. Wir
-            freuen uns sehr, wenn Sie Serlo mit einer Spende unterstützen.
+          <p className="mb-8">
+            Jeden Monat nutzen über 1.2 Millionen Schüler*innen und Lehrkräfte
+            Serlo.
           </p>
-          <Button href="/spenden">
-            Spenden <FontAwesomeIcon icon={faArrowCircleRight} size="1x" />
-          </Button>
-        </FlexCol>
-      </Section>
-    </>
+          <ButtonWithWings
+            className="serlo-new-landing-button inline"
+            href="/serlo"
+          >
+            Mehr über uns
+          </ButtonWithWings>
+        </div>
+      </AboutSection>
+
+      <CommunityWall />
+
+      <section className="mt-20 mb-20 mx-side">
+        <img src="/_assets/img/landing/birds.svg" className="mx-auto" />
+        <h3
+          style={{ hyphens: 'auto' }}
+          className={clsx(
+            'text-center text-4xl font-bold',
+            'leading-cozy tracking-tight',
+            'max-w-2xl mt-7 mx-auto'
+          )}
+        >
+          Zusammen setzen wir uns für mehr Bildungsgerechtigkeit und die
+          digitale Transformation unserer Schulen ein.
+        </h3>
+      </section>
+
+      <PartnerSection className="text-center">
+        <h3
+          className={clsx(
+            'text-center text-4xl font-bold',
+            'leading-cozy tracking-tight',
+            'max-w-2xl mt-32 mx-auto relative z-10 mb-16'
+          )}
+        >
+          Partner und Förderer
+        </h3>
+        <PartnerListNew />
+        <ButtonWithWink
+          className={clsx(
+            'hidden md:inline-block mx-auto mt-12',
+            'font-bold text-xl rounded-lg text-truegray-700 ',
+            'px-8 py-4 tracking-tight border-truegray-700 border-solid border-2',
+            'hover:border-brand-light hover:no-underline hover:text-brand-light'
+          )}
+          href="/partner"
+        >
+          Alle Partner ansehen
+        </ButtonWithWink>
```

Comment:
This button is not visible in mobile nor desktop version due to the class "hidden". Can it be deleted?

Reply:
`hidden` uses `display: none` which is then overwritten by `md:inline-block`.